_posts/2023-04-04-segment-replication.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+title:  "Reduce compute costs and increase throughput with segment replication, available GA in OpenSearch 2.7."
+authors:
+- satnandi
```

Comment:
Please include Marc from my team, since he contributed and made a lot of this content.